api-reference/10 UI Components/dxDropDownBox/1 Configuration/items.md

```diff
@@ -1,10 +1,22 @@
 ---
 id: dxDropDownBox.Options.items
-hidden: 
+type: Array<Object>
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+An array of items displayed by the UI component.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The **items** array can contain:
+
+- Objects with fields described in this section
+- Objects with any other fields. In this case, specify the [itemTemplate](/api-reference/10%20UI%20Widgets/DataExpressionMixin/1%20Configuration/itemTemplate.md '{basewidgetpath}/Configuration/#itemTemplate').
```

Comment:
Since this doc member relates only to the DDBox property, you shouldn't use {basewidgetpath} and {widgetName}(line 18).